src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -365,7 +366,15 @@ internal int GetHashCodeNoParameters()
         internal override DiagnosticInfo? GetUseSiteDiagnostic()
         {
             DiagnosticInfo? info = null;
-            CalculateUseSiteDiagnostic(ref info);
+            if (CallingConvention.IsCallingConvention(CallingConvention.ExtraArguments) ||
+                CallingConvention.IsCallingConvention(CallingConvention.FastCall))
+            {
+                info = new CSDiagnosticInfo(ErrorCode.ERR_UnsupportedCallingConvention, ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat));
```

Comment:
>info = new CSDiagnosticInfo(ErrorCode.ERR_UnsupportedCallingConvention, ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat)); [](start = 16, length = 135)

We probably still want to do the ```else``` part and then merge. I believe some errors are higher in priority than others.